shell/middleware/authenticated.js

```diff
@@ -146,41 +143,8 @@ function invalidResource(store, to, redirect) {
 }
 
 export default async function({
-  route, app, store, redirect, $cookies, req, isDev, from, $plugin, next
+  route, store, redirect, from, $plugin, next
 }) {
-  if ( route.path && typeof route.path === 'string') {
-    // Ignore webpack hot module reload requests
-    if ( route.path.startsWith('/__webpack_hmr/') ) {
-      return;
-    }
-
-    // Ignore the error page
-    if ( route.path.startsWith('/fail-whale') ) {
-      return;
-    }
```

Comment:
This code was necessary because we didn't want to perform authentication for the fail-whale page but the fail-whale page was using the home template/layout. I made it so fail-whale can render on it's own without a template/layout.